kubetest/e2e.go

```diff
@@ -393,6 +436,8 @@ type deployer interface {
 	IsUp() error
 	SetupKubecfg() error
 	Down() error
+	FederationUp() error
```

Comment:
Also if federation deployment is relevant to all deployments, that makes it sound like it should not be part deployment interface?